pom.xml

```diff
@@ -455,6 +464,11 @@
                 <artifactId>browserstack-local-java</artifactId>
                 <version>${version.browserstack-local-java}</version>
             </dependency>
+            <dependency>
+                <groupId>com.splunk</groupId>
+                <artifactId>splunk</artifactId>
+                <version>${version.splunk}</version>
+            </dependency>
```

Comment:
Hi, my two cents (I talked about it with Pavel, so he should have more context)
- splunk indexing is asynchronous 
- splunk dependency should be optional
- there should be strict split between api and implementation
- we should not expect any particular fields in events (eg. level, namespace)

Reply:
Thanks for your input. I talked to Pavel and the comments regarding the raised concerns are below:

- splunk indexing is asynchronous
  If there is a latency which is likely case, we will have to wait before we conduct the search.

- splunk dependency should be optional
  OK

- there should be strict split between api and implementation
  OK

- we should not expect any particular fields in events (eg. level, namespace)
  Ok, field preselection was removed. Splunk API will return all the fields that it provides.